src/styles/base/_override.icons.scss

```diff
@@ -17,3 +17,12 @@ svg {
     }
   }
 }
+
+.ds-c-dialog__close {
+  background: url(./images/CloseSymbol.svg) center center no-repeat !important;
+  background-size: 16px 16px !important;
+  padding: 5px !important;
+  min-width: 20px;
+  min-height: 20px;
+  color: transparent !important;
+}
```

Comment:
This may be a good pr to add a new `_override.dialog.scss` file (under src/styles/components folder):
- move this new override code there? and 
- add `@import "override.dialog";` to src/styles/components/_index.scss.
- then also move the below `ds-c-dialog` from `src/styles/components/_override.other.scss` to this new file too?
```-
-.ds-c-dialog {
-  outline: 0;
-}```